Source/BoundaryConditions/PEC_Insulator.cpp

```diff
@@ -142,10 +142,16 @@ namespace
                                 amrex::Real const rmirror = ijk_mirror[idim] + 0.5_rt*(1._rt - is_nodal[idim]);
                                 // Calculate radial scale factor
                                 sign *= rmirror/rguard;
+#if defined(WARPX_DIM_RSPHERE)
```

Comment:
Could you add a comment here, explaining why this needs to be done, for spherical geometry?

Reply:
This block of code is unneeded and has been removed.